✨ Add boilerplate page (#9)
HackMort/jdk-workflow: src/content/docs/en/proyects-boilerplate.md

---

> title: "Accelerate Your Development with the JDK Boilerplate"

Reviewer: Change to:

```suggestion
title: "JDK Boilerplate"
```

> description: "Discover the power of the JDK Boilerplate in streamlining your development process. This comprehensive blog post guides you through cloning the repository, installing dependencies, and running the project. Learn about essential commands, notes, customizations, and components, as well as utilizing Astro and VS Code extensions for enhanced efficiency."

Reviewer: Change to:
```suggestion
description: "Take a look at our Boilerplate to speed up our development process. This page will guide you through all the steps like: cloning the repository, installing dependencies, and running the project. Learn about essential commands, notes, customizations, and components, as well as utilizing Astro and VS Code extensions for enhanced efficiency."
```

  }
---

1. **Easy Setup and Installation**:
   Clone the JDK Boilerplate repository and quickly set up your project with the provided commands. Learn how to install dependencies and run the project effortlessly.

2. **Convenient Commands and Configurations**:
   Explore the wide range of commands available to streamline your development workflow. From installing dependencies and building your production site to previewing your changes locally, leverage the power of predefined commands for increased productivity.

3. **Customization and Component Creation**:
   Learn how to customize the boilerplate to meet your project requirements. Create components with ease by following step-by-step instructions, and understand how to import styles, configure modal instances, and listen to modal events effectively.

4. **Boosting Efficiency with Astro Extensions**:
   Discover the Astro extension for Visual Studio Code, providing syntax highlighting and convenient code snippets. Install the Astro Snippets extension for quick and easy component creation, saving valuable time during development.

5. **Ensuring Code Quality and Consistency**:

>    Utilize the provided ESLint, Prettier, and Stylelint configurations to maintain code quality and adhere to best practices. Learn about available resources for further reference on configuration options and style rules.

Reviewer: At the end of this paragraph we should have a link to the resources that we mentioned here. Just to make sure they don't leave the site to google what are those things (in case they don't know anything at all about ESlint, Prettier, etc)

> > Visit the <a href="https://github.com/HackMort/jdk-boilerplate/tree/develop" target="_blank">JDK Boilerplate repository</a> and discover how this boilerplate can save you time and effort.

Reviewer: ```suggestion
>  and discover how our boilerplate can save you time and effort.
```
